cinn/runtime/cuda/cuda_util.cc

```diff
@@ -1531,6 +1531,31 @@ void cinn_call_uniform_random(void *v_args, int num_args, float min, float max,
   }
 }
 
+void cinn_call_randint(void *v_args, int num_args, int min, int max, int seed, void *stream) {
+  cinn_pod_value_t *args = static_cast<cinn_pod_value_t *>(v_args);
+  cinn_buffer_t *output  = args[0].operator cinn_buffer_t *();
+  cinn_type_t dtype      = output->type;
+  size_t numel           = output->num_elements();
+
+  curandGenerator_t generator = CurandGenerator::GetInstance().GetGenerator();
+  CURAND_CALL(curandSetStream(generator, static_cast<cudaStream_t>(stream)));
+  if (seed != 0) {
+    CURAND_CALL(curandSetPseudoRandomGeneratorSeed(generator, static_cast<unsigned long long>(seed)));
+  }
```

Comment:
```suggestion
  CurandGenerator::GetInstance().SetSeed(seed);
```
这个API会在seed为0时设置全局seed